### PR TITLE
change default value for tidb_ddl_reorg_worker_cnt and tidb_ddl_reorg_batch_size

### PR DESCRIPTION
**Pull request by tsthght**
### What is changed, added or deleted? (Required)
```
change tidb_ddl_reorg_worker_cnt 's default value from 16 to 4 when the version is 3.0.3
change tidb_ddl_reorg_batch_size 's default value from 1024 to 256 when the version is 3.0.3
```

### Which TiDB version(s) do your changes apply to? (Required)
```
tidb_ddl_reorg_worker_cnt 's default value is 4 when the version is 3.0.3
tidb_ddl_reorg_batch_size 's default value is 256 when the version is 3.0.3
```

- [ ] master (the latest development version, including v4.0 changes for now)
- [ ] v3.1 (TiDB 3.1 versions)
- [✅ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

File: reference/configuration/tidb-server/tidb-specific-variables.md [hunk 1]
```diff
@@ -293,13 +293,15 @@ set @@global.tidb_distsql_scan_concurrency = 10
 ### tidb_ddl_reorg_worker_cnt
 
 - Scope: GLOBAL
-- Default value: 16
+- Default value(before 3.0.3): 16
+- Default value(3.0.3): 4
```
Comment (zimulala) [inline reference/configuration/tidb-server/tidb-specific-variables.md]: (in 3.0.3 or later)

File: reference/configuration/tidb-server/tidb-specific-variables.md [hunk 2]
```diff
@@ -293,13 +293,15 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
 - This variable is used to set the concurrency of the DDL operation in the `re-organize` phase.
 
 ### tidb_ddl_reorg_batch_size
 
 - Scope: GLOBAL
-- Default value: 1024
+- Default value(before 3.0.3): 1024
+- Default value(3.0.3): 256
```
Comment (zimulala) [inline reference/configuration/tidb-server/tidb-specific-variables.md]: (in 3.0.3 or later)

File: reference/configuration/tidb-server/tidb-specific-variables.md (hunk 1 above)
Comment (tsthght) [reply reference/configuration/tidb-server/tidb-specific-variables.md]: thanks, I had fixed it

File: reference/configuration/tidb-server/tidb-specific-variables.md (hunk 2 above)
Comment (tsthght) [reply reference/configuration/tidb-server/tidb-specific-variables.md]: Thanks, I had fixed it.

**Pull request merged**